examples/plot_simulate_beta.py

```diff
@@ -0,0 +1,132 @@
+"""
+===============================
+09. Simulate beta modulated ERP
+===============================
+
+This example demonstrates how event related potentials (ERP) are modulated
+by prestimulus beta events.
+"""
+
+# Authors: Nick Tolley <nicholas_tolley@brown.edu>
+
+###############################################################################
+# Let us import hnn_core
+import hnn_core
+from hnn_core import simulate_dipole, law_model
+from hnn_core.viz import plot_dipole
+
+###############################################################################
+# We begin by instantiating the network model described in Law et al. 2021.
```

Comment:
add a proper reference like in the LFP example?

Reply:
Unfortunately I may be jumping the gun here, the paper is accepted in "Cerebral Cortex" but not published just yet (it should be published by the release). I can add the reference once it has a proper DOI and everything.

Reply:
I see, I think it's also fine if you put the Bioarxiv link for now and update it later. Just so we don't forget